src/main/java/com/kodilla/ecommercee/service/DbSeervice.java

```diff
@@ -14,15 +14,15 @@
 
```

Comment:
Może dobrym pomysłem byłoby żebyśmy nazywali te serwisy zawierając nazwę klasy np. GroupDbService? Bo chyba do każdej klasy będziemy potrzebowali osobnego serwisu

Reply:
A to nie będzie tak, że jak dodamy tutaj wszystkie dao jako final to wtedy poprzez @RequiredArgsConstructor będziemy mogli z nich korzystać?

Reply:
W sumie nie wiem. Wydawało mi się, że jeśli dodamy po kilka metod dla kilku klas to zrobi się z tego bałagan. Ale pewnie możemy też to rozdzielić później w razie potrzeby